index.js

```diff
@@ -152,30 +192,15 @@ function sendNotification(endpoint, params) {
 
       if (vapid && !isGCM) {
         // VAPID isn't supported by GCM.
+        vapid.audience = urlParts.protocol + '//' + urlParts.hostname;
```

Comment:
+1 waiting. 

FWIW, being a push operator and grokking how customers might want to use VAPID, I'd want to be able to specify a URI of my own crafting for "aud", with the default being no value specified. Push server doesn't really care aside from trying to identify the reported source if the "sub" doesn't answer their emails, and as I noted, there's good reason why a subscription service might want to specify a different or seemingly arbitrary value.